src/main/webapp/profileStyle.css

```diff
@@ -134,12 +138,36 @@ a#logout-link i {
   padding: 7px;
 }
 
+#profile-info .insert-new-info {
+  margin-left: 65%;
```

Comment:
This value is too big and because of that the text inputs got out of the form border. Try use ~30% instead.